### DISCUSSION TITLE
New feed doesn't show commits

### DISCUSSION THREAD

**Discussion by spicata** [community/community#66316] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I used to use the old feed to see the commits of the repos that I had watched. With the new feed, this seems to be completely unavailable. This makes it totally useless to me, and probably for others too.

**Comment by hrach** [comment]
This feedback is not new, you had it for the whole time and still ignoring users wanting this. Sad.

**Reply by spicata** [reply]
It's quite the shame
